datasets/arabic_billion_words/README.md

```diff
@@ -44,102 +26,167 @@ task_ids:
   - [Licensing Information](#licensing-information)
   - [Citation Information](#citation-information)
 
-## Dataset Description
+## [Dataset Description](#dataset-description)
 
-- **Homepage:** http://www.abuelkhair.net/index.php/en/arabic/abu-el-khair-corpus
-- **Repository:**
-- **Paper:** https://arxiv.org/pdf/1611.04033
-- **Leaderboard:**
-- **Point of Contact:**[Ibrahim Abu El-Khair](iabuelkhair@gmail.com)
+- **Homepage:** [http://abuelkhair.net/index.php/en/arabic/abu-el-khair-corpus](http://abuelkhair.net/index.php/en/arabic/abu-el-khair-corpus)
+- **Repository:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Paper:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Point of Contact:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- **Download Size:** 3672.46 MB
 
-### Dataset Summary
+### [Dataset Summary](#dataset-summary)
 
 Abu El-Khair Corpus is an Arabic text corpus, that includes more than five million newspaper articles.
 It contains over a billion and a half words in total, out of which, there are about three million unique words.
 The corpus is encoded with two types of encoding, namely: UTF-8, and Windows CP-1256.
 Also it was marked with two mark-up languages, namely: SGML, and XML.
 
-### Supported Tasks and Leaderboards
+### [Supported Tasks](#supported-tasks)
 
-[More Information Needed]
+[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
 
-### Languages
+### [Languages](#languages)
 
-Arabic
+[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
 
-## Dataset Structure
+## [Dataset Structure](#dataset-structure)
 
-### Data Instances
+We show detailed information for up to 5 configurations of the dataset.
 
-[More Information Needed]
+### [Data Instances](#data-instances)
 
-### Data Fields
+#### Alittihad
 
-The data fields are:
-- "url": string, original url of the article,
-- "head_line": string, headline of the article,
-- "date": string, date of the article,
-- "text": string, text content of the article,
+Download size: 332.13 MB
 
-### Data Splits
+An example of 'train' looks as follows.
+```
+{'': 'This example is too large to print.'}
+```
```

Comment:
It would be cool to see a few things even though the examples are too big

For example we can 
Option 1: crop the example instead
````
This example was too big and has been cropped
```
{
    "field1": "lorem ipsum blabla...
    "field2": [{"foo": "lorem ipsum blabla...
}
```
````
Option 2: collapse the example using some markdown and if the users want to read it they have to extend it.
Option 3: a mix of 1 and 2

Reply:
We can do option 1!